libs/platform/ebpf_object.c

```diff
@@ -129,7 +129,7 @@ _ebpf_object_tracking_list_insert(_Inout_ ebpf_core_object_t* object)
 Done:
     ebpf_lock_unlock(&_ebpf_object_tracking_list_lock, state);
 
-    return EBPF_SUCCESS;
+    return return_value;
```

Comment:
Is there a test which causes a failure in this function?

Reply:
Yes, this fails in stress when the object ID table fills up.